README.md

```diff
@@ -1,9 +1 @@
-# backend
-
-## Apollo Studio
-
-In order to have cookies work within Apollo Studio during development, please make sure your Apollo Studio configuration matches the following:
-
-![apollo_config](/docs/apollo_config.png)
-
-Connection settings can be found gear icon near the top left of Apollo Studio.
+# backend
```

Comment:
I think you grabbed the wrong changes cause this removes the updated README.

Reply:
I think my repo for this branch has gone south. Might need to delete this and create another branch